integration/src/test/resources/org/mule/test/components/redelivery-policy-config.xml

```diff
@@ -9,6 +9,15 @@
                            http://www.mulesoft.org/schema/mule/vm http://www.mulesoft.org/schema/mule/vm/current/mule-vm.xsd
                            http://www.mulesoft.org/schema/mule/test-components http://www.mulesoft.org/schema/mule/test-components/current/mule-test-components.xsd">
 
+    <configuration defaultErrorHandler-ref="globalErrorHandler" />
+
+    <error-handler name="globalErrorHandler">
+        <on-error-propagate type="EXPRESSION">
+            <set-payload value="#[error]"/>
```

Comment:
use `#[!null.a]` expression to force an expression error

Reply:
I didn't intend to force an expression error there, just get the error itself for the log. However, I did change it in the two places where I needed to force an expression error.